apps/studio/config-transformer.js

```diff
@@ -0,0 +1,36 @@
+// Putting these to its own file so we can call it from command line.
+
+/**
+ * Transform an object and cast the values to the correct type.
+ *
+ * If the value is a number, cast it to a number. Otherwise, it's a string.
+ **/
+function transform(obj) {
+  const clone = Object.assign({}, obj);
```

Comment:
Hmm, I'm not sure how to do that. We'll need to compile/build the code to be able to use external dependencies from there. 

Currently, I can run the `configTypesBuilder.js` script because everything is vanilla and run using `node`. It's faster this way especially if we're gonna hook it with pre-commit.

It's probably better to do that if we can have a new repo, no?